src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  const basicDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days >= 7) {
+    return days * rentalCost - basicDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
You still have "magic" numbers, let's move them to specific variables

Reply:
Thanks